### ISSUE TITLE
Explain the Iroha versioning

### ISSUE THREAD

**Issue by 6r1d** [hyperledger/iroha-2-docs#297]
[Python 3 guide](https://hyperledger.github.io/iroha-2-docs/guide/python.html#python-3-guide) and [JavaScript guide] mention certain versions of Iroha, currently `v2-pre-rc.1` for Python 3 and `2.0.0-pre-rc.9` for JavaScript.

This will need to be clarified for the new user: we should add links to download a version with a certain tag, explain how to get a certain version from SDK, apply the same approach for the SDKs and tell what SDK code works with a certain Iroha version.

TL;DR: more is needed to tell a version; we need to explain what steps a user needs to perform.

**Comment by outoftardis** [comment]
This is essentially what we discussed re versions and doc testing, right?
